src/App.js

```diff
@@ -30,14 +35,33 @@ class App extends Component {
   goHome = () => {
     this.setState({ movieView: false })
   }
+  
+  formatDate = date => {
+    const splitDate = date.split('-')
+    const newDate = splitDate.slice(1)
+    newDate.push(splitDate[0])
+    return newDate.join('/')
+  }
+
+  sortMovies = (flicks) => {
```

Comment:
Sorting function will likely be more dynamic in the future in order to sort by user preference